### ISSUE TITLE
Wrong Shippable test on type(x).__name__ print

### ISSUE THREAD

**Issue by nerzhul** [ansible/ansible#21127]
##### ISSUE TYPE
 - Bug Report

##### COMPONENT NAME

shippable

##### ANSIBLE VERSION

```
devel
```

##### SUMMARY

Shippable failed to detect type(x) usage correctly. When using type(x).__name__ to retrieve a type, it thinks we are trying to test which type is this.

here is the incorrect build

https://app.shippable.com/runs/589add9b6ee43c0f00b55fe3/1/console

##### STEPS TO REPRODUCE

```
self.module.fail_json(msg='TSIG update error (%s): %s' % (type(e).__name__, str(e)))
```

##### EXPECTED RESULTS

working test

##### ACTUAL RESULTS

```
ERROR: Type comparison using type() found on line 197. Use isinstance() instead
```

**Comment by nerzhul** [comment]
ping @gundalow @mattclay

**Comment by mattclay** [comment]
Resolved by #21162.
